examples/arithmetics/src/language-server/generated/ast.ts

```diff
@@ -126,20 +125,13 @@ export interface ArithmeticsAstType {
     Statement: Statement
 }
 
-export class ArithmeticsAstReflection implements AstReflection {
+export class ArithmeticsAstReflection extends AbstractAstReflection {
 
     getAllTypes(): string[] {
         return ['AbstractDefinition', 'BinaryExpression', 'DeclaredParameter', 'Definition', 'Evaluation', 'Expression', 'FunctionCall', 'Module', 'NumberLiteral', 'Statement'];
     }
 
-    isInstance(node: unknown, type: string): boolean {
-        return isAstNode(node) && this.isSubtype(node.$type, type);
-    }
-
-    isSubtype(subtype: string, supertype: string): boolean {
-        if (subtype === supertype) {
-            return true;
-        }
+    isSubtypeOverride(subtype: string, supertype: string): boolean {
```

Comment:
This generated method should be `protected`. About the name: could `computeIsSubtype` or similar be more understandable?